src/CosmosDB/CosmosDB/help/Update-AzCosmosDBMongoDBUserDefinition.md

```diff
@@ -5,88 +5,74 @@ online version: https://learn.microsoft.com/powershell/module/az.cosmosdb/update
 
-### ByParentObjectDataActionsParameterSet
+### ByParentObjectParameterSet
 ```
-New-AzCosmosDBMongoDBUserDefinition -Id <String> -UserName <String> -Password <String> [-Mechanisms <String>]
- -DatabaseName <String>
+Update-AzCosmosDBMongoDBUserDefinition -Id <String> -UserName <String> -Password <String>
+ [-Mechanisms <String>] -DatabaseName <String>
  -Roles <System.Collections.Generic.List`1[Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.Models.MongoDB.PSMongoRole]>
  [-CustomData <String>] -DatabaseAccountObject <PSDatabaseAccountGetResults>
- [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
-```
-
-### ByParentObjectPermissionsParameterSet
-```
-New-AzCosmosDBMongoDBUserDefinition -Id <String> -UserName <String> -Password <String> [-Mechanisms <String>]
- -DatabaseName <String>
- -Roles <System.Collections.Generic.List`1[Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.Models.MongoDB.PSMongoRole]>
- [-CustomData <String>] -DatabaseAccountObject <PSDatabaseAccountGetResults>
- [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+ [-DefaultProfile <IAzureContextContainer>] [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
 ```
 
-### ByResourceIdParameterSet
+### ByObjectParameterSet
 ```
-New-AzCosmosDBMongoDBUserDefinition -Id <String> -UserName <String> -Password <String> [-Mechanisms <String>]
- -DatabaseName <String>
+Update-AzCosmosDBMongoDBUserDefinition -Id <String> -UserName <String> -Password <String>
+ [-Mechanisms <String>] -DatabaseName <String>
  -Roles <System.Collections.Generic.List`1[Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.Models.MongoDB.PSMongoRole]>
- [-CustomData <String>] [-ResourceId <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [-CustomData <String>] -InputObject <PSMongoDBUserDefinitionGetResults>
+ [-DefaultProfile <IAzureContextContainer>] [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm]
  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-Update an existing CosmosDB MongoDB User Definition.
-In order to specify the Role Definition's Roles,  use the New-AzCosmosDBMongoDBRole cmdlet to create PSMongoRole objects to pass in through the Roles parameter.
+This cmdlet updates an existing MongoDB user definition in a specified Cosmos DB account.
 
 ## EXAMPLES
 
-### Example 1: Default
+### Example 1
 ```powershell
-$Roles = New-AzCosmosDBMongoDBRole -Database test -Role roleName
-
-
-Update-AzCosmosDBMongoDBUserDefinition `
-	-AccountName accountName `
-	-ResourceGroupName resourceGroupName `
-	-DatabaseName 'test' `
-	-UserName 'myName' `
-	-Password 'pass' `
-	-Id id `
-	-Mechanisms 'SCRAM-SHA-256' `
-	-CustomData 'test' `
-	-Roles $Roles 
+$subscriptionId = "bebf1773-330e-4db8-8df9-7b66b8a80868"
+$rgName = "myResourceGroup"
+$AccountName = "myCosmosDBAccount"
+$DatabaseName = "myDatabase"
+$Username1 = "myUser"
+$UserDefinitionId1 = $DatabaseName + "." + $Username1
+$Pass1 = "myPassword"
```

Comment:
Test: can coding guidelines detect sub id and password
cc @vidai-msft

Reply:
Looks like not detected.